hw/drivers/flash/spiflash/src/spiflash.c

```diff
@@ -1033,6 +1035,11 @@ spiflash_erase_sector(const struct hal_flash *hal_flash_dev,
     spiflash_cs_deactivate(dev);
 #endif
 
+    /* Usually erase sector takes around 40ms, let's wait here that time.
+     * TODO: Do it configurable per flash.
```

Comment:
so why not just add syscfg for this now? :-)

Reply:
because I believe correct fix should be to add defines per all the supported flash and not give it to the user to set. Those values are in the specifications.